core/store/src/lib.rs

```diff
@@ -222,28 +220,74 @@ impl StoreUpdate {
         Ok(())
     }
 
-    /// Inserts a new reference-counted value into the database, or adjusts its
-    /// reference count if it is already there.
+    /// Inserts a new reference-counted value or increases its reference count
+    /// if it’s already there.
     ///
-    /// It is a programming error if `update_refcount` supplies a different
-    /// value than the one stored in te database. Use it for rc columns.
-    pub fn update_refcount(&mut self, column: DBCol, key: &[u8], value: &[u8], rc_delta: i64) {
+    /// It is a programming error if `increase_refcount_by` supplies a different
+    /// value than the one stored in the database.  It may lead to data
+    /// corruption or panics.
+    ///
+    /// Panics if this is used for columns which are not reference-counted
+    /// (see [`DBCol::is_rc`]).
+    pub fn increase_refcount_by(
```

Comment:
Std calls these increment/decrement rather than increase/decrease, slight preference for sticking with std terminology

https://doc.rust-lang.org/stable/alloc/sync/struct.Arc.html#method.increment_strong_count